cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -604,42 +631,146 @@ For Workflows, you can also retrieve:
 
-   * - HTTP Method
-     - ``GET <base-url>/namespaces/default/apps/PurchaseHistory/services/CatalogLookup/runs?status=completed&limit=1``
-   * - Description
-     - Retrieve the most recent successful completed run of the Service *CatalogLookup* of the Application *PurchaseHistory*
-   * - Returns
-     - ``[{"runid":"cad83d45-ecfb-4bf8-8cdb-4928a5601b0e","start":1415051892,"end":1415057103,"status":"STOPPED"}]``
-   * - 
-     - 
-   * - HTTP Method
-     - ``GET <base-url>/namespaces/default/apps/PurchaseHistory/workflows/PurchaseHistoryWorkflow/schedules``
-   * - Description
-     - Retrieves the schedules of the Workflow *PurchaseHistoryWorkflow* of the Application *PurchaseHistory*
-   * - Returns
-     - ``[{"schedule":{"name":"DailySchedule","description":"DailySchedule with crontab 0 4 * * *","cronEntry":"0 4 * * *"},``
-       ``"program":{"programName":"PurchaseHistoryWorkflow","programType":"WORKFLOW"},"properties":{}}]``
-   * - 
-     - 
-   * - HTTP Method
-     - ``GET <base-url>/namespaces/default/apps/PurchaseHistory/workflows/PurchaseHistoryWorkflow/nextruntime``
-   * - Description
-     - Retrieves the next runtime of the Workflow *PurchaseHistoryWorkflow* of the Application *PurchaseHistory*
-   * - Returns
-     - ``[{"id":"DEFAULT.WORKFLOW:developer:PurchaseHistory:PurchaseHistoryWorkflow:0:DailySchedule","time":1415102400000}]``
 
+.. container:: table-block-example
+
+  .. list-table::
+     :widths: 99 1
+     :stub-columns: 1
+
+     * - Example: Retrieving A Schedule
+       - 
+       
+  .. list-table::
+     :widths: 15 85
+     :class: triple-table
+
+     * - Description
+       - Retrieves the schedules of the Workflow *PurchaseHistoryWorkflow* of the Application *PurchaseHistory*
+      
+     * - HTTP Method
+       - ``GET <base-url>/namespaces/default/apps/PurchaseHistory/workflows/PurchaseHistoryWorkflow/schedules``
+         
+     * - Returns
+       - | ``[{"schedule":{"name":"DailySchedule","description":"DailySchedule with crontab 0 4 * * *","cronEntry":"0 4 * * *"},``
+         | `` "program":{"programName":"PurchaseHistoryWorkflow","programType":"WORKFLOW"},"properties":{}}]``
+         
+
+.. container:: table-block-example
+
+  .. list-table::
+     :widths: 99 1
+     :stub-columns: 1
+
+     * - Example: Retrieving The Next Runtime
+       - 
+       
+  .. list-table::
+     :widths: 15 85
+     :class: triple-table
+
+     * - Description
+       - Retrieves the next runtime of the Workflow *PurchaseHistoryWorkflow* of the Application *PurchaseHistory*
+      
+     * - HTTP Method
+       - ``GET <base-url>/namespaces/default/apps/PurchaseHistory/workflows/PurchaseHistoryWorkflow/nextruntime``
+         
+     * - Returns
+       - | ``[{"id":"DEFAULT.WORKFLOW:developer:PurchaseHistory:PurchaseHistoryWorkflow:0:DailySchedule","time":1415102400000}]``
+       
+
+Schedules and Workflows: Suspend and Resume
```

Comment:
Schedules and Workflows are different and so there suspend/resume operations. Its better to put the suspend/resume for the schedule in the building block page for `Schedules`?